pom.xml

```diff
@@ -110,21 +115,27 @@
         <version.lib.freemarker>2.3.23</version.lib.freemarker>
         <version.lib.helidon>0.9.0</version.lib.helidon>
         <version.lib.junit>5.1.0</version.lib.junit>
-        <version.lib.maven>3.3.9</version.lib.maven>
+        <version.lib.hamcrest>1.3</version.lib.hamcrest>
+        <version.lib.maven>3.6.2</version.lib.maven>
```

Comment:
Will this require a higher version of maven, or will it still work with older maven versions?

Reply:
It should still work; I didn't change my version of maven (3.5.2).